aspnetcore/includes/blazor-preview-notice.md

```diff
@@ -0,0 +1,6 @@
+> [!NOTE]
+> **Blazor is an unsupported experimental web framework. Blazor is in preview.**
+>
+> Blazor preview documentation is provided to assist you in trying out Blazor. Blazor isn't a committed Microsoft product at this time. Blazor's feature set, capabilities, and tooling are subject to change at any time without notice. Blazor shouldn't be used for production workloads.
```

Comment:
One more nit - I would remove "at any time without notice" as we do try to announce when we make breaking changes and you can always see the planned changes on GitHub.